data-access-layer/eventmesh/LockManager.js

```diff
@@ -199,11 +221,15 @@ class LockManager {
 
   _getLockType(operation) {
     if (_.includes(CONST.APISERVER.WRITE_OPERATIONS, operation)) {
-      return CONST.ETCD.LOCK_TYPE.WRITE;
+      return CONST.APISERVER.LOCK_TYPE.WRITE;
     } else {
-      return CONST.ETCD.LOCK_TYPE.READ;
+      return CONST.APISERVER.LOCK_TYPE.READ;
     }
   }
+  _getLockTTL(operation) {
+    /* jshint unused:false */
+    return 10000; // Shall be fetched from config
```

Comment:
Will have to get it from config.

Reply:
Didn't understand the above? Will the same be updated in this PR itself?